pom.xml

```diff
@@ -1201,4 +1201,34 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>tests-with-dependencies</id>
```

Comment:
This is a nice improvement. However, shouldn't it be available in all packages which include benchmarks? Would it make sense to move the common logic (profile ID, activating variable, maven-assembly-plugin's settings to the top-level pom)?

Reply:
It is available for all the modules.

`presto-geospatial` module was just an example in a sample command `./mvnw clean install -pl presto-geospatial -DcreateTestsWithDependenciesJar`

But you can run the command for any module you want.

Reply:
@arhimondr Andrii, thanks for explaining. I didn't notice that this change was in top-level POM file. :-)